src/ranged.cpp

```diff
@@ -1576,9 +1578,8 @@ static void cycle_action( item &weap, const tripoint &pos )
     const auto mag = weap.magazine_current();
     if( mag && mag->type->magazine->linkage ) {
         item linkage( *mag->type->magazine->linkage, calendar::turn, 1 );
-        if( weap.gunmod_find( itype_brass_catcher ) ) {
-            linkage.set_flag( "CASING" );
```

Comment:
Why is this line removed?

Reply:
Previously, this was used to find them and add the to inventory/ground when removing the brass catcher since they were not containers. Now the items are directly put into the brass catcher and removing it gives a filled brass catcher that doesn't need additional evaluation, so from my understanding it's unnecessary.

Reply:
I see, thanks.